label_studio_converter/converter.py

```diff
@@ -510,6 +532,7 @@ def create_child_node(doc, tag, attr, parent_node):
             root_node = doc.documentElement
             create_child_node(doc, 'folder', output_image_dir_rel, root_node)
             create_child_node(doc, 'filename', image_name, root_node)
+            create_child_node(doc, 'annotator', item['completed_by'].get('email', 'none'), root_node)
```

Comment:
the same as with COCO

Reply:
Besides the option to create one file per annotator, we might use [this node](https://gist.github.com/Prasad9/30900b0ef1375cc7385f4d85135fdb44#file-annotation-xml-L5)